daemon/daemon_unix.go

```diff
@@ -184,6 +184,20 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *runconfig.HostC
 		hostConfig.CpusetCpus = ""
 		hostConfig.CpusetMems = ""
 	}
+	cpusAvailable, err := sysInfo.IsCpusetCpusAvailable(hostConfig.CpusetCpus)
+	if err != nil {
+		return warnings, fmt.Errorf("Invalid value %s for cpuset cpus.", hostConfig.CpusetCpus)
```

Comment:
it looks like these errors are good. I think it would be good to add them to the new errors package directly.